vermouth/processors/merge_chains.py

```diff
@@ -42,8 +42,15 @@ def merge_chains(system, chains):
         The system to modify.
     chains: list[str]
         A container of chain identifier.
+    all_chains:
     """
-    chains = set(chains)
+    if not all_chains:
+        chains = set(chains)
+    else:
+        l = []
+        for molecule in system.molecules:
+            l.append([node.get('chain') for node in molecule.nodes.values()][0])
+        chains = set(l)
```

Comment:
```suggestion
        chains = set()
        for molecule in system.molecules:
            chains.update(node.get('chain') for node in molecule.nodes)  # Molecules can contain multiple chains
```
Do we need to deal with `None` and `''` chains?